### PR TITLE
Jenkins: update integration test for CAPM3 v1a4 release

### PR DESCRIPTION
**Pull request by maelk**
This is needed to take the new CAPM3_VERSION variable into use

### CODE REVIEW

File: jenkins/scripts/files/run_integration_tests.sh [hunk 1]
```diff
@@ -2,18 +2,16 @@
...
+VARS_FILE="${1}"
+GITHUB_TOKEN="${2}"
+
+source "${VARS_FILE}"
+
+export CAPI_VERSION
+export CAPM3_VERSION
+export IMAGE_OS
+export DEFAULT_HOSTS_MEMORY
+export NUM_NODES
```
Comment (fmuyassarov) [inline jenkins/scripts/files/run_integration_tests.sh:14]: Can we avoid having these vars exported here by adding `export` within `"${CI_DIR}/files/vars.sh"`

Comment (maelk) [reply jenkins/scripts/files/run_integration_tests.sh:14]: I'd rather keep the two separated. It is clearer to export the variables where needed, than in another script that should only care about setting the variables, not how they are used.

**Comment by fmuyassarov** [comment]
> /test-integration

```
release_tag=null
+++ [[ null == \n\u\l\l ]]
+++ echo 'Failed to get the release tag for https://api.github.com/repos/metal3-io/cluster-api-provider-metal3/releases/latest'
+++ exit 1
++ export 'CAPM3RELEASE=Failed to get the release tag for https://api.github.com/repos/metal3-io/cluster-api-provider-metal3/releases/latest'
```

**Comment by maelk** [comment]
Once https://github.com/metal3-io/metal3-dev-env/pull/400 ismerged, this should pass. We are hitting the API rate limit of github

**Comment by maelk** [comment]
@fmuyassarov lgty otherwise ? I'll just retrigger the test when https://github.com/metal3-io/metal3-dev-env/pull/400 is merged, and then we can proceed with merging this one and the last one in Nordix jjb repo ?

**Comment by fmuyassarov** [comment]
> @fmuyassarov lgty otherwise ? I'll just retrigger the test when [metal3-io/metal3-dev-env#400](https://github.com/metal3-io/metal3-dev-env/pull/400) is merged, and then we can proceed with merging this one and the last one in Nordix jjb repo ?

Sure, sounds good to me!

**Pull request merged**